src/transformers/generation_utils.py

```diff
@@ -628,32 +646,29 @@ def _get_decoder_start_token_id(self, decoder_start_token_id: int = None, bos_to
 
     @staticmethod
     def _expand_inputs_for_generation(
-        input_ids: torch.LongTensor,
         expand_size: int = 1,
         is_encoder_decoder: bool = False,
-        attention_mask: Optional[torch.LongTensor] = None,
-        encoder_outputs: Optional[ModelOutput] = None,
```

Comment:
these input arguments were never used directly, but rather as part of the unpacked `model_kwargs`, so there is no need to have them